### PR TITLE
ex_7_ch3.py

### PR DESCRIPTION
**Pull request by amymhaddad**
**Note: this is also a practice in converting floating numbers to integers.

Instructions: Create a program that calculates the area of a room. Prompt the user for the length and width of the room in feet. Then display the area in both square feet and square meters.

Example: 
What is the length of the room in feet? 15
       What is the width of the room in feet? 20
       You entered dimensions of 15 feet by 20 feet.
       The area is
       300 square feet
       27.871 square meters

### CODE REVIEW

File: ex_7_ch3.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+length = int(input("What is the length of the room in feet? "))
+width = int(input("What is the width of the rrom in feet? "))
+
+print(f"You entered dimensions of {length} feet by {width} feet.")
+
+square_feet = length * width
+print(f"The area of the room is {square_feet} square feet.")
+
+square_meters = square_feet * 0.092903
+print(f"The area of the room is {int(square_meters)} square meteres.")
```
Comment (paulghaddad) [inline ex_7_ch3.py]: `square_meters` is already an `int` (see line 17). Thus, you don't have to convert it to an `int` here. You can just put `square_meters` inside the curly braces.

Comment (paulghaddad) [inline ex_7_ch3.py]: Nit-pick: you misspelled `meters`.

File: ex_7_ch3.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (paulghaddad) [inline ex_7_ch3.py]: You have a bunch of blank lines at the bottom. I would avoid unnecessary lines.

**Comment by amymhaddad** [comment]
@paulghaddad I updated this exercise.

**Comment by amymhaddad** [comment]
@paulghaddad I updated this exercise.

File: ex_7_ch3.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+SQ_FT_TO_METERS = 0.092903
```
Comment (paulghaddad) [inline ex_7_ch3.py]: Perfect!

**Pull request merged**